.github/renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":dependencyDashboard"
+    ],
+    "enabledManagers": [
+        "gomod",
+        "github-actions"
+    ],
+    "gomod": {
+        "packageRules": [
```

Comment:
what do these pacakgeRules do?

Reply:
Group dependency updates, so that it doesn't create a PR per go.mod